packages/kit/src/cli.js

```diff
@@ -38,7 +38,12 @@ prog
 prog
 	.command('sync')
 	.describe('Synchronise generated files')
-	.action(async () => {
+	.option(
+		'--mode',
+		'Specify a mode for loading environment variables (default: "development")',
+		undefined
```

Comment:
```suggestion
		'--mode',
		'Specify a mode for loading environment variables',
		'development'
```

Reply:
We can do this, but "development" as the default is also defined at the `sync` level, so it's technically unnecessary, right?